modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/adapter/SW360ReleaseClientAdapter.java

```diff
@@ -150,4 +151,13 @@ public SW360Release createRelease(SW360Release releaseFromArtifact, boolean uplo
         }
         return Optional.empty();
     }
+
+    public List<SW360SparseAttachment> getAttachmentsOfRelease(SW360Release release, HttpHeaders header) {
```

Comment:
For my understanding, this only returns the list of available attachments, right? Or does it also download the attachments, if so, my comment from below is valid here as well.

Reply:
It only gets references to the attachment. For a download another call must be made